UserCard.jsx

```diff
@@ -0,0 +1,33 @@
+function UserCard(props) {
+  const [avatar, setAvatar] = React.useState('Loading...')
+  const [firstName, setFirstName] = React.useState('Loading...')
+  const [lastName, setLastName] = React.useState('Loading...')
+  const [email, setEmail] = React.useState('Loading...')
+
+  React.useEffect(() => {
+    async function getUsers(userId) {
```

Comment:
👍 to making a named function for this fetching. I think this works well too. You could also define this outside the component entirely and then use a `then` callback to do the setting of state, but either way works!